src/core/AutoRest.Core/Properties/Resources.resx

```diff
@@ -252,4 +252,10 @@
   <data name="InvalidConstraint" xml:space="preserve">
     <value>Constraint is not supported for this type and will be ignored.</value>
   </data>
+  <data name="OperationParametersNotAllowedMessage" xml:space="preserve">
+    <value>Parameters "subscriptionId" and "api-version" are not allowed in the operations section of azure swagger specs</value>
```

Comment:
`Parameters "subscriptionId" and "api-version" are not allowed in the operations section` ?